Makefile

```diff
@@ -183,6 +183,16 @@ CEPH_IMG       ?= quay.io/ceph/demo:latest
 # NNCP
 NNCP_INTERFACE ?= enp6s0
 
+# Ceilometer
+CEILOMETER_IMG                 ?= quay.io/openstack-k8s-operators/ceilometer-operator-index:main-latest
```

Comment:
:latest was pushed to the repository as of yesterday, so I think we can change it to match it

Reply:
Oh no, the -index:latest was not generated, only the ceilometer-operator. So :latest does not exist for this image and currently it is called :main-latest